more_autograding_examples/verilog_hello_world/config/config.json

```diff
@@ -0,0 +1,44 @@
+{
+    "required_capabilities": "default",
+    "grading_parameters": {
+        "TOTAL_POINTS": 10,
+        "AUTO_POINTS": 10,
+        "TA_POINTS": 0,
+        "EXTRA_CREDIT_POINTS": 0
+    },
+    "autograding_method": "docker",
+    "autograding": {
+        // compile .v files
+        "submission_to_compilation": ["**/*.v"],
+        // run a.out
+        "compilation_to_runner": ["**/a.out"]
+    },
+    // use the verilog container for grading
+    "container_options": {
+        "container_image": "tkoz0/submitty-verilog"
```

Comment:
I think this needs to be changed to "tkoz0/submitty-verilog:latest" as there is a need to add this custom docker to the "Docker UI" section which will only accept a docker with the correct tag <Image name>:<tag>. And currently there is no way of manipulating the alias of the docker added.